Answers.md

```diff
@@ -1,5 +1,14 @@
 1.  Explain the differences between `client-side routing` and `server-side routing`.
+    Client side routing takes place within the web browser and typically does not need to consult a server at all or multiple times. Server side routing needs a constant connection to a server and makes frequent requests
```

Comment:
This isn't untrue, but it kind of misses the big picture. The characteristic that most distinguishes client-side-routing from server-side-routing is that it loads everything that it needs on the first load, and then only loads again if absolutely needs to. This make the initial load time generally longer, but subsequent load times are faster. Server side is the opposite.